can-validate-validatejs.md

```diff
@@ -1,21 +1,73 @@
 @module {Object} can-validate-validatejs
 @parent can-ecosystem
-@group can-validate-validatejs.core 1 can-validate-validatejs Core
 @package ./package.json
 
-@description A plugin for CanJS that wraps any validation library to `can.validate`.
-**Can-Validate doesn't do any validation of its own** but instead provides some abstraction to your library of choice. The chosen library is registered with can-validate using a shim.
+@description Create [can-validate/types/validator] functions using [validate.js](https://validatejs.org/).
 
-@type {Object}
+@signature `makeValidator(constraints)`
+
+Returns a validator function that can be used to validate a single value.
+
+@param {Object} constraints An object of definitions used by the Validate.js library to run validations on a value.
+
+@return {Validator} A validator type function that takes a value and validates it against the provided constraints.
+
+@signature `makeValidator.many(constraints)`
```

Comment:
This should be a property on the `makeValidator` function, not documented in here.